tests/utils.go

```diff
@@ -149,8 +149,6 @@ func init() {
 }
 
 func FlagParse() {
```

Comment:
Now the method is not doing parsing any more, consider changing its name to something like - `FillUnprovidedFlags` or `NormalizeFlags`?

Reply:
Done